airgun/views/host.py

```diff
@@ -154,6 +154,22 @@ def is_displayed(self):
             self.title, exception=False) is not None
 
 
+class InheritButton(GenericLocatorWidget):
```

Comment:
Isn't `Button()` object provide everything you need? Like whether it is active and etc?

It has for example:
```
 @property
    def active(self):
        return 'active' in self.browser.classes(self)
```

I know that you have some specific, but code is really looks redundant